### ISSUE TITLE
emit cert as Java KeyStore 

### ISSUE THREAD

**Issue by aep** [edgelesssys/marblerun#307]
### Use case

java requires using a "keystore" instead of just pem encoded certs.
so we'll have to emit those from premain.

### Describe your solution

(https://github.com/pavlo-v-chernykh/keystore-go) could be used to provide the nessesary template func

something like

```
"Marbles": {
        "javathing": {
            "Parameters": {
                "Files": {
                    "/app/keystore.jks": "{{ javaKeyStore .MarbleRun.MarbleCert.Cert }}",
```

**Comment by daniel-weisse** [comment]
I would prefer not to add language specific encodings to the manifest.
In my opinion, parsing files is something that should be left to the application itself.

If PEM is something your application does not handle well, you can also use `raw`, to get the certificate in DER encoded certificate, or `hex`, to get the DER certificate hex encoded. See [the docs](https://docs.edgeless.systems/marblerun/#/workflows/define-manifest?id=templates) for more details.

Tagging @m1ghtym0 in case he has a different opinion.

**Comment by m1ghtym0** [comment]
I agree with Daniel that ideally, encodings should be language neutral.
However, I see the problem here and it's just another encoding.
IMO, that's not a big deal and we can have as many encodings as necessary, especially in hindsight of lift&shift.
So, @aep please feel free to add JavaKeyStore as a file encoding.
